and-bible/app/src/main/res/values/strings.xml

```diff
@@ -268,6 +268,77 @@ TODO: error and download_document_confirm_prefix need to be parameterized rather
+
+    <string name="conf_speak_auto_bookmark">Automatic bookmarking</string>
+    <string name="restore_settings_from_bookmarks">Save playback settings to bookmarks</string>
+
+    <string name="conf_speak">Speak and play earcon sounds for</string>
+    <string name="conf_replace_divinename">Replace divine name with Yahweh</string>
+
+    <string name="conf_change_chapter">Chapter changes</string>
+    <string name="conf_change_title">Titles</string>
+    <string name="conf_speak_footnotes">Footnotes</string>
+
+    <string name="conf_sleep_timer">Set sleep timer</string>
+    <string name="sleep_timer_title">Sleep time in minutes</string>
+    <string name="sleep_timer_timer_set">Set sleep timer. Timer set: %d minutes</string>
+
+    <string name="conf_playback_settings">Playback settings</string>
+    <string name="conf_speak_settings">General speak settings</string>
+    <string name="conf_speak_bookmarking_settings">Bookmarking settings</string>
+    <string name="sleep_timer_started">Sleep timer started, %d minutes</string>
+
+    <string name="speak_next">Next</string>
+    <string name="speak_previous">Previous</string>
+    <string name="widget_toggle_sleep_timer">Toggle sleep timer</string>
+    <string name="bookmarkwidget_helptext">Speak bookmarks shortcut buttons will appear here</string>
+    <string name="sleep_timer_active_at">Sleep timer: %s minutes left.</string>
+    <string name="speak_status_stopped">- stopped - </string>
+    <string name="speak_bookmark_help">Speak bookmark help</string>
+    <string name="speak_bookmark_help_message"><b>Automatic bookmarking</b>
+        \nAuto add bookmark when pausing and stopping. Automatically generated bookmarks are
+        labelled with special Speak label. If you start on Speak-labeled bookmark, bookmark will
+        be moved to the new verse location when you stop / pause next time. If there is Speak label
+        and also other labels on bookmark, only Speak label will be removed and bookmark will stay.
+        \n
+        \n<b>Save playback settings to bookmarks</b>
+        \nThere are three <i>playback settings</i>: Speak (chapter changes / titles),
+        Play earcon sounds for (chapter changes / titles) and Speak speed.
+        If this option is enabled, those settings are saved to Speak bookmarks.
+        \n
+        \n<i>Usage example:</i> Let us say that you want to listen to Genesis and Romans. You want to listen to
+        Genesis with speak speed 150%, and without interruptions between chapters, but when listening to
+        Romans you want to have 90% speak speed and hear audible earcon sound when chapter changes. You can have
+        independent playback settings for these books, when these settings are stored to bookmarks.
+    </string>
+    <string name="speak_bookmarks_menu_title">Speak from bookmark</string>
+
+    <string-array name="divinename_original">
+        <item>Lord</item>
```

Comment:
Actually I am not sure about these divine name changes.  We are essentially changing the translation.  Could there be legal issues?  People choose a Bible translation and that involves all of the decisions relating to divine names.  Should we also allow some people to change the gender words so instead of reading 'man' it reads 'man and woman'.  I don't think divine name mapping adds much and it complicates the code so I would like to remove it.

Reply:
I love also this feature, and the intention of having capital letters (`<divineName>` tag) in these texts is to make it easy to recognize original tetragrammaton in the text (Yahweh). When we are using text to speech, we can't see changes in font, so this recognition could be only implemented by:

 - making some adjustment to the speech (pitch, volume, etc.)
 - making yet another earcon sound
 - or just simply pronouncing divineNames differently, i.e. by saying "Yahweh" (or something else, if so desired) instead of "Lord"

I like the third approach the most. It is clear and rather simple solution and works well at least in English and Finnish (and most likely with other languages too, but depending on languages, it might require mapping several words instead of one/two).

This feature is disabled by default and we are not rendering divineNames differently on bible displays -- just TTS if this feature is enabled. I'm not completely sure about possible legal issue with this, but my best guess is that there should be no problem with this. I welcome corrections here and I can always remove this feature later if it is found illegal.